data/tabular/MUV_466/meta.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: MUV_466
+description: Activity in the MUV_466 assay
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+targets:
+    - id: MUV-466
+      type: boolean
+      description: MUV-466
+      names:
+          - noun: an agonist of the S1P1 receptor
+license: CC BY 4.0
+links:
+    - url: https://academic.oup.com/nar/article/44/D1/D1075/2502602?login=false
+      description: corresponding publication
+    - url: https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/muv.csv.gz
+      description: Data source
+num_points: 14841
+bibtex:
+    - |
+      @article{doi:10.1021/ci8002649,
+      author = {Rohrer, Sebastian G. and Baumann, Knut},
+      title = {Maximum Unbiased Validation (MUV) Data Sets for Virtual Screening Based on PubChem Bioactivity Data},
+      journal = {Journal of Chemical Information and Modeling},
+      volume = {49},
+      number = {2},
+      pages = {169-184},
+      year = {2009},
+      doi = {10.1021/ci8002649},
+      URL = {https://doi.org/10.1021/ci8002649}}
+templates:
+    - The {#molecule|compound|chemical|molecular species|chemical compound!} with the {SMILES__description} {#representation of |!}{SMILES#} is {MUV-466#not
+      &NULL}an agonist of the S1P1 receptor
```

Comment:
Instead of `an agonist of the S1P1 receptor` you could use `{MUV-466__names__noun}`.

Reply:
We can also add instruction tuning and chat text templates. But this is not too critical.

Reply:
> Instead of `an agonist of the S1P1 receptor` you could use `{MUV-466__names__noun}`.

it was an automated process here when creating the templates.